### ISSUE TITLE
[Bug] The video was overwritten because of the same name

### ISSUE THREAD

**Issue by yuioto** [kingwingfly/fav#13]
RT.
If fav fetch has the same value in Title, video overwriting will occur.

## To Reproduce
Omit initialization and login.

```shell
fav track BV15L4y1v7KG
fav track BV1XU4y1E7xi
fav fetch
fav pull
```

## Expected behavior

There should be videos `BV15L4y1v7KG` and `BV1XU4y1E7xi` locally.

However, there is only one video, and because the file names are the same, the files processed later overwrite the files processed first.

## Information

OS: ArchLinux
Output of `rustc -V`: rustc 1.75.0 (82e1608df 2023-12-21) (Arch Linux rust 1:1.75.0-2)

However, this information is not relevant to this error.

**Comment by yuioto** [comment]
https://github.com/kingwingfly/fav/discussions/10#discussioncomment-8409712

**Comment by kingwingfly** [comment]
fixed: 5ab46b4256d055317b34c8d6f97fd4e861c73a2e on v0.2.2

Repeated resource will be added a bvid.
